src/main/java/frc/robot/commands/auto/AutoRoutines.java

```diff
@@ -86,7 +79,9 @@ public void registerCommandsToPathplanner() {
          NamedCommands.registerCommand("elevatorIntake",new SetElevator(Config.ElevatorSetPoints.FEEDER).withTimeout(2));
 
         // NamedCommands.registerCommand("coralIntake", new CoralIntake(-0.3,  0.3).withTimeout(1.5));
-         NamedCommands.registerCommand("CoralScore", new CoralDepositorCommand(true, false).withTimeout(2));
+         NamedCommands.registerCommand("CoralScore", new CoralDepositorCommand(true, false).withTimeout(1));
+         NamedCommands.registerCommand("reset", PhotonSubsystem.getInstance().getResetCommand());
+         NamedCommands.registerCommand("vision-move",new PhotonMoveToTargetLeft(false, false, false));
```

Comment:
PhotonMoveToTargetLeft() is using the subsystem PhotonSubsystemLeftReef. 
PhotonMoveToTarget() is using the subsystem PhotonSubsystem.
Reset the corresponding subsystem in line 83.